css/properties/fill.json

```
@@ -10,12 +10,14 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
```

```suggestion
              "version_added": "1"
```

See: https://github.com/caugner/css-properties/blame/d158e8e43aad53776f4f5a278cb14d948d99bfaf/properties.css#L154